data/_ui/interfaces.txt

```diff
@@ -525,6 +529,55 @@ interface "plugins"
 
 
 
+interface "install plugins"
+	sprite "ui/plugins panel"
+		center -20 -20
+	button c "_Controls"
+		center -300 -230
+		dimensions 90 30
+		color medium
+		hover bright
+	button s "_Settings"
+		center -300 -190
+		dimensions 90 30
+		color medium
+		hover bright
+	button p "_Plugins"
+		center -300 -150
+		dimensions 90 30
+		color bright
+
+	active if "can install/update"
+	button i "_Install/Update"
```

Comment:
This button should read just Install if the plugin isn't installed, and just Update if it is installed already *and* out-of-date.

Reply:
ok